app/components/authorization-code/login-redirect/login-redirect.component.js

```diff
@@ -15,7 +15,31 @@ import Controller from './login-redirect.controller';
 const LoginRedirectComponent = {
   controller: Controller,
   template: `
-    <a href="" ng-click="$ctrl.login()" data-se="login-link">Login with Okta</a>
+    <p>
```

Comment:
Just a suggestion: Moving the HTML to its own file and using template URL can help simplify this JS file and we can also take advantage of the editor to format and write HTML files faster. This can also help in maintaining things later.